backend/seed.ts

```diff
@@ -160,6 +231,56 @@ async function main() {
     },
   })
 
+  // Add comment to Jon's post
+  await db.post.update({
+    where: {
+      id: 2,
+    },
+    data: {
+      threads: {
+        create: [
+          {
+            startIndex: 15,
+            endIndex: 38,
+            highlightedContent: 'Les Pluies de Castamere',
+            comments: {
+              create: [
+                {
+                  body: 'You bastard!',
+                  author: {
+                    connect: { id: tywin.id },
+                  },
+                },
+                {
+                  body: 'HA HA!',
+                  author: {
+                    connect: { id: ned.id },
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+  })
```

Comment:
@Lanny I have a wee bit of a strange situation when rendering this post in the app. It loads fine, and if I inspect the DOM I can see the comment thread appears to have been correctly applied around the selection, but we don't get the highlight for some reason 🧐. Any ideas?

Reply:
hmm, that's odd. Does it have the expected class name? `highlightedContent` doesn't seem to line up with the indices, how did you get the `startIndex` and `endIndex` values?

Reply:
The indices should be correct. I actually got them by directly console.logging the `startIndex` and `endIndex` values while creating a text selection manually. It did appear to have the correct class name